python/pfs/drp/stella/centroidLines.py

```diff
@@ -158,7 +158,7 @@ def convolveImage(self, exposure):
             Convolved image.
         """
         psf = exposure.getPsf()
-        sigma = psf.computeShape().getTraceRadius()
+        sigma = psf.computeShape(psf.getAveragePosition()).getTraceRadius()
```

Comment:
Has the API changed for ``psf.computeShape()``? This change is not mentioned in the commit comment.